deployments/scripts/protobuf-codegen

```diff
@@ -18,9 +18,10 @@ pushd "${repo_root}/proto/"
 # Make sure the lockfile is up to date.
 buf dep update penumbra
 
-# Pull our vendored IBC-go proto defs so we can get reflection for service definitions.
+# Pull our vendored cosmos/IBC proto defs so we can get reflection for service definitions.
 # The penumbra dependencies will override some of these.
-buf export buf.build/penumbra-zone/ibc-go-vendor --output rust-vendored/
+buf export buf.build/cosmos/cosmos-sdk:e7a85cef453e4b999ad9aff8714ae05f --output rust-vendored/
+buf export buf.build/cosmos/ibc --output rust-vendored/
```

Comment:
In https://github.com/penumbra-zone/penumbra/blob/a9d5c7b5fcb5188804695c79144d4afe660fee05/proto/penumbra/buf.yaml#L7 we have some version-locking of these same deps, as they are included in the penumbra protos. Can we add some version-pinning here? As written, it looks like the `buf.build/cosmos/ibc` dep will pull from latest every time this script is run, which may not be what we want: controlled updates for regenerating the protos makes the most sense.

Unfortunately, there's not a good semver system, so in the past I've been inspecting hashes manually to pin deps.